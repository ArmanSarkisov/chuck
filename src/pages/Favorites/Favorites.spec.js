import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Favorites } from './index';

describe('Favorites page test', () => {
  let component;
  beforeEach(() => {
    component = mount(<Favorites />);
  });

  it('should render Favorite component', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
