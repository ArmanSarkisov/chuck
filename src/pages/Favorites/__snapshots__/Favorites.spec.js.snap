// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favorites page test should render Favorite component 1`] = `
<Favorites>
  <div
    className="pt-10"
  >
    <h2
      className="text-center text-2xl"
    >
      You dont have favorite jokes
    </h2>
  </div>
</Favorites>
`;
